### PR TITLE
fuse:bump to 3.11.0

### PR DESCRIPTION
**Pull request by lhw828**

### CODE REVIEW

**Comment by lunar-linux-b01** [comment]
SUCCESS!

File: filesys/fuse/BUILD [hunk 1]
```diff
@@ -1,10 +1,4 @@
-OPTS+=" --enable-lib \
-        --enable-util \
-        --disable-static \
-        --disable-example \
-        --sbindir=/usr/bin"
-
-default_build &&
+default_meson_build &&
 
 if module_installed systemd; then
```
Comment (Ratler) [inline filesys/fuse/BUILD:3]: We can drop the if check for systems, since that would always be present.

That init file should always be removed :)

**Review by Ratler — CHANGES_REQUESTED** [review]

**Comment by lunar-linux-b01** [comment]
SUCCESS!

**Comment by stumbles** [comment]
There are two fuse modules in other. Fuse and fuse-3 (currently 3.11.x). They are not the same. Last I tried a while back locally, using fuse at version 3.11.x caused some dependent modules not to work. Its been a while since testing this, so before bumping please thoroughly check dependent modules will still work.

**Pull request closed without merging**